### PR TITLE
Polishes header and registration swiper

### PR DESCRIPTION
**Pull request by sophiabersani**
Stories:
https://www.pivotaltracker.com/story/show/120607799
https://www.pivotaltracker.com/story/show/120608355
https://www.pivotaltracker.com/story/show/120623691

Swiper Component:
- Adds the “Swiper” 3rd party component as a custom component for ease of updates.
- Updates the Swiper to support custom dots for pagination.
- Fixes 2 bugs related to “shouldContinue” method.

Polish:
- Reduces the size of the Header in views that did not need it (Registration/Login).
- Sets the banner/logo as a static component to match mock-ups.
- Adds custom dots for registration pagination.
- Adjusts styling on Login view's inputs.

Bug Fixes:
- Fixes bug with backwards button in Terms of Service page.
- Fixes bug with swiping backwards in registration page.

### CODE REVIEW

File: js/components/Header.js [hunk 1]
```diff
@@ -30,40 +32,33 @@ const Header = React.createClass ({
...
       let title = this.state.title
-      let routeStack = nextProps.navState.routeStack
+      let path = this.state.path
+      let routeStack = nextProps.navigator.getCurrentRoutes()
       let position = routeStack.length - 1
       let nextTitle = routeStack[routeStack.length-1].title
-      if (nextTitle && nextTitle !== title) {
-        title = nextTitle
-      }
+      let nextPath = routeStack[routeStack.length-1].path
```
Comment (jgoley) [inline js/components/Header.js]: I think lines 37 - 39 could be refactored. maybe by removing `position` and declare `route=routeStack[routeStack.length-1]` so we can avoid repeating it later.

Comment (jgoley) [inline js/components/Header.js]: Lines 19 and 20 could use the same refactor

**Comment by jgoley** [comment]
@gbersani is it possible to remove the whitespace shown in the screenshot below so that the blue region is the beginning of the overlap?

screen shot 2016-06-01 at 3 30 06 pm

File: js/components/Header.js (hunk 1 above)
Comment (sophiabersani) [inline js/components/Header.js]: Both good points. Will look into that.

**Comment by sophiabersani** [comment]
@jgoley Should be possible, it seems that the swiper is missing a few pixels on the render.
Will submit a fix soon.

**Comment by sophiabersani** [comment]
Done.

**Pull request merged**